guides/common/modules/proc_importing-a-repository.adoc

```diff
@@ -29,7 +25,7 @@ total 68M
 . On {ProjectServer} where you want to import, create/enable a repository with the same name and label as the exported content.
 . In the {ProjectWebUI}, navigate to *Content* > *Products*.
 . Click the *Yum content* tab.
-. Add the same `Yum` content that the exported Content View version includes.
+. Add the same `Yum` content that the exported content includes.
```

Comment:
```suggestion
```
Unnecessary. Import will automatically set that up. User does not need to be doing this step.